source/templates/actions.md

```diff
@@ -167,7 +167,7 @@ You can specify an alternative event by using the `on` option.
 You should use the normalized event names [listed in the View guide][1].
 In general, two-word event names (like `keypress`) become `keyPress`.
 
-[1]: /guides/understanding-ember/the-view-layer/#toc_adding-new-events
+[1]: http://emberjs.com/api/classes/Ember.View.html#toc_event-names
```

Comment:
In the View Guide, the only documentation I see about handling events is this:
http://guides.emberjs.com/v1.11.0/views/handling-events/

Which links to this:
http://emberjs.com/api/classes/Ember.View.html#toc_event-names

```
You should use the normalized event names listed in the View guide. In general, two-word event names (like keypress) become keyPress.
```